CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 ## [Unreleased]
 
+## [2.14.0] - 2018-09-27
+
+### Added
+- Credhub introduced as a user-accessible component (independent of future cf-deployment requirements)
+- Exposed SMTP variable to allow for account creation & password reset
```

Comment:
Why not specify the exact name of the new variable, and where it is set (in UAA)?

Just looked it up, and it is not one, but a while bunch of variables: https://github.com/SUSE/uaa-fissile-release/commit/382b70b0e30a465f7c355bf377f2afbfb0881394, but `SMTP_HOST` and `SMTP_FROM_ADDRESS` might be just the minimum that need to be set, depending on the server.

Reply:
Revised to: `Exposed SMTP_HOST & SMTP_FROM_ADDRESS variables to allow for account creation & password reset`